### ISSUE TITLE
comments-indentation fails for Bash comments

### ISSUE THREAD

**Issue by jgwinner** [adrienverge/yamllint#449]
[comments-indentation] comment not indented like content

this error is generated for the following GitHub actions code, which seems valid to me? I mean, if I change the indent it'll mess up the run command. 

```
    - name: Print service stack outputs
      env:
        $CLOUDFORMATIONOUT: ${{ toJson(steps.erp-create.outputs) }}
      run: |
         echo "$CLOUDFORMATIONOUT"
         ENVIRONMENT=`echo $REPO | tr "/" "-"`
         echo "Environment name: CLOUDFORMATIONOUT"
         echo "::set-output name=cloudformationout::$CLOUDFORMATIONOUT"
         #        echo "${{ steps.erp-create.outputs }} "
         # {
<yamllint fails here> - the line after the commented block

```

**Comment by andrewimeson** [comment]
What version of yamllint? Can you post a complete and valid YAML file that demonstrates the issue?

This passes on the current released version (1.26.3) as well as the tip of master:

```yaml
---
on: push  # yamllint disable-line rule:truthy
jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3

      - name: Print service stack outputs
        env:
          $CLOUDFORMATIONOUT: ${{ toJson(steps.erp-create.outputs) }}
        run: |
           echo "$CLOUDFORMATIONOUT"
           ENVIRONMENT=`echo $REPO | tr "/" "-"`
           echo "Environment name: CLOUDFORMATIONOUT"
           echo "::set-output name=cloudformationout::$CLOUDFORMATIONOUT"
           #        echo "${{ steps.erp-create.outputs }} "
           # {

      - name: More stuff
        run: |
           whatever
```

**Comment by jgwinner** [comment]
Ack, me as well - now. Here's what I think happened.

My github will sometimes convert line endings to CRLF. I've caught some files occasionally with mixed line endings. So I think the above got me, with yamllint interpreting the LF lines following a CR line incorrectly.

Not yamllint's fault - sorry for that!

It drives me crazy when code decides to "help" me by screwing up my files. Every IDE I've used for the last 20 years handles LF just fine. `</rant>`

If it pops back up I'll double check everything and re-open if needed, but for now the exact file that was complaining, as well as your example, scans fine.
